### PR TITLE
Create basic signup page - limited functionality

### PR DESCRIPTION
**Pull request by NickPapayiannakis**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<%= @user.last_name %>
+<%= @user.email %>
+<%= @user.username %>
+<%= @user.password %>
+<%= @user.password_confirmation %>
+<%= @user.height %>
+<%= @user.weight %>
+<%= @user.activities %>
+<%= @user.created_at %>
+<%= @user.updated_at %>
```
Comment (bpassos-zz) [inline app/views/users/show.html.erb:22]: Are these form fields?
If is, dont they need to be wrapped in `<%= form_user do %>` where `form_user` is a helper. ?

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/* mixins, variables, etc. */
```
Comment (bpassos-zz) [inline app/assets/stylesheets/custom.scss:1]: Not sure thins file is the best place to put this stuff. But know we dont have a better place right, now.  Lets keep this in mind, for best CSS architecture.

File: .byebug_history [hunk 3]
```diff
@@ -0,0 +1,7 @@
+continue
```
Comment (bpassos-zz) [inline .byebug_history:1]: Should ```.buybug_history``` be .gitignored ?

**Review by bpassos-zz — COMMENTED** [review]
Left some comments. OK with it merging now, but lets discuss the points raised. 🙌

File: app/assets/stylesheets/custom.scss (hunk 2 above)
Comment (NickPapayiannakis) [reply app/assets/stylesheets/custom.scss:1]: yep, completely agree. It's a stub and can be destroyed once we have proper CSS architecture.

**Pull request merged**